autodepgraph/node.py

```diff
@@ -27,8 +28,15 @@ def __init__(self, name, verbose=False):
                            parameter_class=ManualParameter,
                            vals=vals.Numbers(min_value=0))
 
-        self.add_parameter('dependencies',
-                           docstring='a list of names of Calibration nodes',
+        self.add_parameter('parents',
+                           docstring='List of names of nodes '
+                                     + 'on which this node depends',
+                           initial_value=[],
+                           vals=vals.Lists(vals.Strings()),
+                           parameter_class=ManualParameter)
+        self.add_parameter('children',
+                           docstring='List of names of nodes '
+                                     + 'which depend on this node',
                            initial_value=[],
                            vals=vals.Lists(vals.Strings()),
                            parameter_class=ManualParameter)
```

Comment:
I would also make this a getable only command with reference on how to set it in the docstring.